apps/full-site-editing/full-site-editing-plugin/site-editor-experiment/index.php

```diff
@@ -0,0 +1,162 @@
+<?php
+/**
+ * Site Editor experiment file.
+ *
+ * @package A8C\FSE
+ */
+
+namespace A8C\FSE;
+
+/**
+ * Enables/Disables site editor experiment per blog sticker.
+ */
+function conditionally_enable_site_editor() {
```

Comment:
My thought would be to wrap this function in the `switch_to_blog` like we do here:

https://github.com/Automattic/wp-calypso/blob/8ba6796028f2d54ade7d828c021ff6042b4991b1/apps/full-site-editing/full-site-editing-plugin/dotcom-fse/helpers.php#L40-L58

So basically switch at the beginning if we need to, and then restore at the end of we need to. That way all of the other functions this one calls are also in the correct blog context

Reply:
Yeah, even after wrapping everything in that to ensure the proper blog context, it seems it does not load the editor until I also go enable the experiment on the public-api. 🤔   Even if those functions aren't being called when used in the public-api's context...